atala-prism-sdk/src/commonTest/kotlin/io/iohk/atala/prism/walletsdk/prismagent/PrismAgentTests.kt

```diff
@@ -391,4 +407,59 @@ class PrismAgentTests {
         assert(x is OutOfBandInvitation)
         assert((x as OutOfBandInvitation).type == ProtocolType.Didcomminvitation)
     }
+
+    @Ignore("Work in progress")
+    @Test
+    fun test_anoncreds() = runTest {
+        val fromDID = DID("did:prism:asdf42sf")
+        val toDID = DID("did:prism:asdf42sf")
+
+        polluxMock.extractedCredentialFormatFromMessageReturn = CredentialType.ANONCREDS_REQUEST
+        plutoMock
+
+        val agent = PrismAgent(
+            apollo = apolloMock,
+            castor = castorMock,
+            pluto = plutoMock,
+            mercury = mercuryMock,
+            pollux = polluxMock,
+            connectionManager = connectionManager,
+            seed = null,
+            api = ApiMock(HttpStatusCode.OK, "{\"success\":\"true\"}"),
+            logger = PrismLoggerMock()
+        )
+
+        val attachmentDescriptor =
+            AttachmentDescriptor(
+                mediaType = CredentialType.ANONCREDS_REQUEST.type,
+                data = AttachmentBase64(
+                    ""
+                )
+            )
+        val attachmentID = attachmentDescriptor.id
+
+        val offerCredential = OfferCredential(
+            body = OfferCredential.Body(
+                credentialPreview = CredentialPreview(
+                    attributes = arrayOf(
+                        CredentialPreview.Attribute("", "", "")
+                    )
+                )
+//                formats = arrayOf(
+//                    CredentialFormat(
+//                        attachId = attachmentID,
+//                        format = CredentialType.ANONCREDS.type
+//                    )
+//                )
+            ),
+            attachments = arrayOf(attachmentDescriptor),
+            thid = "1",
+            from = fromDID,
+            to = toDID
+        )
+
+        val invitation = agent.prepareRequestCredentialWithIssuer(did = toDID, offer = offerCredential)
+//        val message = json.decodeFromString<Message>(invitationString)
+        println("stop")
+    }
```

Comment:
This test is not correct or is it?

Reply:
Finished this and added another one to test issued credential.